src/calculateRentalCost.js

```diff
@@ -1,10 +1,20 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
+const SHORT_TERM = 3;
+const SHORT_TERM_DISCOUNT = 20;
+const LONG_TERM = 7;
+const LONG_TERM_DISCOUNT = 50;
+
+const basePrice = 40;
+
 function calculateRentalCost(days) {
-  // write code here
+  let total = days * basePrice;
+
+  if (days >= LONG_TERM) {
+    total -= LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    total -= SHORT_TERM_DISCOUNT;
```

Comment:
The current logic applies a fixed discount for short-term and long-term rentals. Typically, discounts are applied as a percentage of the total cost rather than a flat amount. Consider changing the discount to be a percentage of the total rental cost to better reflect common discount practices.